homeassistant/components/cast/helpers.py

```diff
@@ -197,6 +197,8 @@ async def _fetch_playlist(hass, url):
     try:
         session = aiohttp_client.async_get_clientsession(hass, verify_ssl=False)
         async with session.get(url, timeout=5) as resp:
+            if resp.content_type and resp.content_type.startswith("audio"):
```

Comment:
realized this won't work

https://en.wikipedia.org/wiki/PLS_(file_format)

PLS has type audio/x-scpls

Reply:
yeah, I guess let's just drop this because we already only fetch if URL ends in a certain way anyway.